test/reviewsPagination.spec.js

```diff
@@ -0,0 +1,50 @@
+
+import request from "supertest";
+import { describe, expect, test } from "@jest/globals";
+import runApp from "../src/server.js";
+import { getReviews } from "../src/api.js";
+
+const app = runApp({ getReviews });
+
+let id = 3;
+let page = 1;
+let pageSize = 5;
+
+describe("get correct response format of reviews", () => {
+  
+  test("get max 5 reviews per page ", async () => {
+    const res = await request(app).get(
+      `/api/movies/${id}/reviews?page=${page}&pageSize=${pageSize}`
+    );
+
```

Comment:
Overall it looks good but I think you want to avoid any API requests in your tests. If the API is taken down your test won't work, that's why we want to use mock data instead. If you just create a mock with the same data as the request then your test will still run no matter what, And you will also know exactly what gets put into the test.

Reply:
Ok thanks for the info. I will fix that

Reply:
I think it's fine doing integration test with requests to our API. But yeah I agree there maybe should be some sort of dependency injection so CMS API can be simulated/mocked 👍

Reply:
Thanks for the feedback. I have updated the test